app/views/taxons/show.html.erb

```diff
@@ -1,108 +1,176 @@
 <%= display_header title: page.title, breadcrumbs: [:taxons, page.taxon] %>
 
+<table class="state-table add-bottom-margin">
+  <thead class="text-muted">
+    <tr>
+      <th>
+        state
+      </th>
+      <th>
+        phase
+      </th>
+      <th>
+        tagged
+      </th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>
+        <span class="label state-label--<%= page.publication_state_name %>">
+          <%= page.publication_state_name %>
+        </span>
+      </td>
+      <td>
+        <span class="label phase-label--<%= page.taxon.phase %>">
+          <%= page.taxon.phase %>
+        </span>
+      </td>
+      <td>
+        <span class="label label-default">
+          <%= page.tagged.count %>
+        </span>
+      </td>
```

Comment:
The layout of these spans seems to involve the whitespace on webkit browsers (e.g. Chromium), but doesn't seem to affect Firefox, meaning that the spans appear to touch.

Reply:
Updated in https://github.com/alphagov/content-tagger/pull/659/commits/26b58d4de4247eff6f84fb19d9f7b26df4a0e906